js/plugin/popups.ts

```diff
@@ -345,7 +346,7 @@ class BrowserModPopup extends LitElement {
                           slot="actionItems"
                           title=${icon.title ?? ""}
                           @click=${() => { this.blur(); this._icon_action(index)} }
-                          class=${icon.class ?? ""}
+                          class=${ifDefined(icon.class)}
```

Comment:
adjusting popupcode as well so class not added when class is not defined.

Reply:
Updated to use class map. As class(es) will be a string, but `classMap = { classname: true }` I used a mapObject function, also filtering string split to remove empty strings which split returns when split character not found (empty string or string with 1 word).

Reply:
Yeah. I’m happy to revert.